### PR TITLE
feat: SIGINT-855 coverity local analysis readme

### PR DESCRIPTION
**Pull request by lsynopsys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,17 +133,18 @@ steps:
...
+| `BRIDGE_COVERITY_CONNECT_PROJECT_NAME`  | Project name in Coverity                                                                                                                                                                                                                                                                      | Mandatory     |
+| `BRIDGE_COVERITY_CONNECT_STREAM_NAME`   | Stream name in Coverity                                                                                                                                                                                                                                                                       | Mandatory     |
+| `BRIDGE_COVERITY_LOCAL`                 | If enabled, Synopsys Extension will install the ‘Coverity Thick Client’ on the local system in order to execute the scan. 'coverity_install_directory' can also be given to keep the files for future scans.                                                                                   | Optional     |
```
Comment (sig-tithi) [inline README.md:143]: 'coverity_install_directory' can also be given to keep the files for future scans.  - which files we are referring to?

Comment (lsynopsys) [reply README.md:143]: It will try to download coverity thick client and same files can be kept under this dir. It is nothing but caching of Coverity thick client

Comment (sig-tithi) [reply README.md:143]: 'coverity_install_directory' can also be given to keep the files for future scans- can you rephrase the sentence.
Mention supported values and default value for the same.

**Comment by sig-tithi** [comment]
closing this MR as changes will be done directly in bridge doc
